src/views/admin/manage/registration.tpl

```diff
@@ -50,9 +50,9 @@
 								<!-- IF users.ipSpam -->
 								<i class="fa fa-times-circle text-danger" title="[[admin/manage/registration:list.ip-spam, {users.spamData.ip.frequency}, {users.spamData.ip.appears}]]"></i>
 								<!-- ELSE -->
-								    <!-- IF users.spamChecked -->
-        							<i class="fa fa-check text-success"></i>
-    								<!-- ENDIF users.spamChecked -->
+                                    <!-- IF users.spamChecked -->
+                                    <i class="fa fa-check text-success"></i>
+                                    <!-- ENDIF users.spamChecked -->
```

Comment:
@julianlam i am not sure why it's indented that way :/ it looks fine in my editor 

i'm gonna try something

![screen shot 2019-01-25 at 3 59 38 pm](https://user-images.githubusercontent.com/1398375/51750265-3feccc00-20ba-11e9-9d9b-25aace2912ad.png)

Reply:
You must have a different tab width set up, and also you have spaces instead of tabs.

Reply:
shouldn't the tab width be set by editorconfig https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#tab_width

Reply:
I don't think i have spaces, my editor is following editorconfig